pyperformance/data-files/benchmarks/bm_2to3/vendor/pyproject.toml

```
@@ -0,0 +1,14 @@
+[build-system]
+requires = ["setuptools>=61"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lib2to3"
+version = "3.12"
+description = "lib2to3 for Python 3.13+"
```

```suggestion
description = "lib2to3 for Python 3.13+"
requires-python = ">=3.13.0a0"
license = {text = "PSF-2.0"}
classifiers = [
    "License :: OSI Approved :: Python Software Foundation License",
]
```

Upon further thought, there's a couple bits of additional metadata that you should also include in the `pyproject.toml`, specifically the license stuff (discussed in the top-level comment) and `requires-python`; this will avoid it accidentally being installed on a lower Python version than 3.12, either through a bug, user error, or if this were to ever get distributed elsewhere (by you or someone else) in the future.